website/env.example

```diff
@@ -1,7 +1 @@
-# https://www.algolia.com/apps/XSSC1LRN4S/explorer/browse/Dagger_docs
-APPLICATION_ID=XSSC1LRN4S
-# Copy your "Write API KEY" from https://www.algolia.com/account/api-keys/all?applicationId=XSSC1LRN4S
-API_KEY=
-# This will not work on an ARM Mac, the algolia/docsearch-scraper image was only built for linux 64bit
-# I am using a Docker Engine on Linux here:
-DOCKER_HOST=ssh://192.168.1.22
+REACT_APP_AMPLITUDE_ID=
```

Comment:
@slumbering Can we add a link as a comment above where we get this value from? E.g.

```
# https://analytics.amplitude.com/blocklayer/settings/projects/333523/general
REACT_APP_AMPLITUDE_ID=
```

Reply:
Do you know what link we should use? I'm not sure which one you had in mind.

Reply:
Actually I tried to find where the amplitude api key but couldn't find it. It was Friday, I'll see that on Monday.

Reply:
Found it, it was the **Early access** app.